src/docs/markdown/caddyfile/directives/tls.md

```diff
@@ -359,22 +359,31 @@ Obtains certificates using the ACME protocol. Note that `acme` is a default issu
 
 	- **any_common_name** <span id="any_common_name"/> is a list of one or more common names; Caddy will choose the first chain that has an issuer that matches with at least one of the specified common names.
 
+<aside class="tip">
+
+The `acme` issuer module will implicitly use [ZeroSSL's ACME endpoint](https://zerossl.com/documentation/acme/) (and generate EAB credentials) if you specify the the [`email` global option](/docs/caddyfile/options#email).
```

Comment:
What if instead, we add a sentence to the `zerossl` section that clarifies: "The ZeroSSL API is distinct from its ACME endpoint." or something like that.

Reply:
While that'd cover the distinction more directly, the `acme` section still claims the module is "using Let's Encrypt" by default. For consistency either all or none of the possible implicit/default configuration should be mentioned IMO.

Especially since the [release notes](https://github.com/caddyserver/caddy/releases/tag/v2.8.0) claim that when the `email` global is configured,

> you don't have to make any changes and you'll still get Let's Encrypt and ZeroSSL automatically as defaults.

which, turns out, is arguably not true if `acme` is explicitly configured. That's not apparent (to me anyway) when said configuration is not providing a `directory_url` as an argument nor field value. (For context, I use the `acme` issuer directive to override the DNS resolvers used for challenges (since those aren't globally configurable AFAIK) and had the false intuition it'd leave the dirs intact.)

Reply:
What's the status on this at this point?

Reply:
I think we're still waiting for a few tweaks to be made; but if not, I might try to wrap this up myself at some point. Although, I don't think I have push rights to the branch.